### DISCUSSION TITLE
Par2 and Par2turbo, do both need to be installed?

### DISCUSSION THREAD

**Discussion by DynaMight1124** [OneCDOnly/sherpa#278] [Q&A] [answered]
Hi

I currently have par2 and par2turbo installed, I am wondering if I can safely remove par2 without any packages having issues? I mostly use NZBGet but also have SABnzbd (which I saw used turbo and might had been the reason it was recently installed)

I dont mind leaving both installed if required but obviously no point having both if one isnt needed or could cause conflicts. 

Thanks

**Comment by OneCDOnly** [comment, marked as answer]
Hi @DynaMight1124. :)

No, you only need one installed. sherpa will try to install both, but one will always fail due to incompatible arch.

`Par2` and `Par2turbo` are both used only by SABnzbd. If `Par2turbo` is installed on your NAS, you can remove the `Par2` QPKG. 

Unfortunately, this requirement for a new `Par2turbo` QPKG came out at the same time I contracted COVID19. At the time, sherpa hadn't been coded to handle alternative QPKG requirements, and I opted for the lazy fix: try to install both, and always allow one to fail. 

Now that I've recovered, and have returned to developing, it's an issue I'm working-on. When complete, sherpa will only try to install one or the other.

**Reply by DynaMight1124** [reply]
Thanks for coming back to me and I hope you're feeling better now

**Comment by OneCDOnly** [comment]
Should be good-to-go now. sherpa 231119 will only install either `Par2turbo` (preferred), or `Par2` (if required), but not both. :)
